items/MAXED_ENCHANT_BOOK.json

```diff
@@ -1,47 +1,67 @@
+  "nbttag": "{ench:[0:{lvl:5s,id:32s},1:{lvl:3s,id:8s},2:{lvl:3s,id:5s},3:{lvl:1s,id:6s}],overrideMeta:1b,HideFlags:254,display:{Lore:[0:\"§aThis is to test custom enchant color.\",1:\"\",2:\"§9§d§lBank V, §9Aiming V\",3:\"§9Angler VI, §9Aqua Affinity I\",4:\"§9Blessing V, §9Caster VI\",5:\"§9Cleave V, §9Critical VI\",6:\"§9Cubism V, §9Depth Strider III\",7:\"§9Dragon Hunter V, §9Efficiency V\",8:\"§9Ender Slayer VI, §9Execute V\",9:\"§9Experience IV, §9Feather Falling X\",10:\"§9Fire Aspect II, §9First Strike IV\",11:\"§9Flame I, §9Fortune III\",12:\"§9Frail VI, §9Giant Killer VII\",13:\"§9Growth VII, §9Harvesting V\",14:\"§9Impaling III, §9Infinite Quiver X\",15:\"§9Knockback II, §9Lethality VI\",16:\"§9Life Steal IV, §9Looting IV\",17:\"§9Luck VI, §9Luck of the Sea VI\",18:\"§9Lure VI, §9Magnet VI\",19:\"§9Piercing I, §9Power VII\",20:\"§9Protection VII, §9Punch II\",21:\"§9Rainbow I, §9Rejuvenate V\",22:\"§9Replenish I, §9Respiration III\",23:\"§9Scavenger V, §9Sharpness VII\",24:\"§9Snipe III, §9Spiked Hook VI\",25:\"§9Sugar Rush III, §9Telekinesis I\",26:\"§9Thorns III, §9Thunderlord VI\",27:\"§9True Protection I, §9Vampirism VI\",28:\"§9Venomous VI\",29:\"\",30:\"§c§l§ka§r §c§l§c§lSPECIAL §c§l§ka\"],Name:\"§f§f§cMax Enchanted Book\"},ExtraAttributes:{rarity_upgrades:1,originTag:\"CRAFTING_GRID_SHIFT\",id:\"MAXED_ENCHANT_BOOK\",enchantments:{impaling:3,thorns:3,luck:6,feather_falling:10,respiration:3,knockback:2,ender_slayer:6,scavenger:5,experience:4,fire_aspect:2,luck_of_the_sea:6,life_steal:4,thunderlord:5,rejuvenate:5,sharpness:7,lethality:5,efficiency:5,cleave:5,frail:6,execute:5,aqua_affinity:1,rainbow:1,true_protection:1,infinite_quiver:10,angler:6,replenish:1,blessing:5,critical:6,piercing:1,protection:6,telekinesis:1,lure:6,giant_killer:6,punch:2,caster:6,sugar_rush:3,harvesting:5,dragon_hunter:5,venomous:5,spiked_hook:6,power:7,cubism:5,fortune:3,looting:4,snipe:3,vampirism:6,magnet:6,depth_strider:3,first_strike:4,ultimate_bank:5,growth:6,flame:1,aiming:5},uuid:\"d4fe1393-3b18-4c43-921b-a6d2770a3444\",timestamp:\"7/25/20 9:58 AM\"},AttributeModifiers:[]}",
   "damage": 0,
   "lore": [
     "§aThis is to test custom enchant color.",
```

Comment:
Have you tested if lore this long displays ingame?

Reply:
http://prntscr.com/wmp1km

Reply:
wait i send you the file on dc so you can test it